lib/firecord/record.rb

```diff
@@ -0,0 +1,72 @@
+module Firecord
+  module Record
```

Comment:
Firecord::Record has initialize method

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Module-Initialize.md)